### ISSUE TITLE
Unclear license header in dist/onfido.min.js

### ISSUE THREAD

**Issue by tapdiego-amzn** [onfido/onfido-sdk-ui#679]
Hi — https://github.com/onfido/onfido-sdk-ui/blob/master/dist/onfido.min.js contains the following license header:
```
/*!
  Copyright (c) 2015 Jed Watson.
  Based on code that is Copyright 2013-2015, Facebook, Inc.
  All rights reserved.
*/
```
I’m finding this header confusing. Could you please clarify how it applies to your code?
I did some research and it seems the header might apply to a minified, modified version of https://github.com/JedWatson/exenv/blob/master/index.js included in the file.

**Comment by stefaniacardenas** [comment]
Hi, I think it's due to the fact that we use `classnames` library and they enforce the copyright to be included in the production build. I will investigate if there is a way of making sure that this header is only displayed with other vendors packages.

**Comment by tapdiego-amzn** [comment]
Thanks — it does seem though like this header comes from `exenv` rather than `classnames`; the text is copied verbatim from its `index.js` file. So I guess `exenv` requires the header to be copied as well?

**Comment by kopijunkie** [comment]
Hi @tapdiego-amzn 

Yes, that is correct. Both of Jed Watson's libraries have a similar requirement for the copyright inclusion. As Stefania's mentioned, we're looking into cleaning up the minified file by only having it displayed with the other vendors' package.

**Comment by kopijunkie** [comment]
Hi @tapdiego-amzn 

We've released 6.2.0 earlier this week that includes the PR above that separates the 3rd party licenses from the main bundle. I'll close this ticket for now but feel free to re-open if you have any further questions.

**Comment by tapdiego-amzn** [comment]
Thank you!
